### PR TITLE
OKTA-573953 : SIW V3 missing no-translate attributes

### PR DESCRIPTION
**Pull request by leemchale-okta**
## Description:

This PR adds the missing `no-translate` attributes in SIW V3.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-573953](https://oktainc.atlassian.net/browse/OKTA-573953)

### CODE REVIEW

File: src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap [hunk 1]
```diff
@@ -1709,14 +1709,14 @@ exports[`authenticator-verification-select-authenticator renders form 1`] = `
                         class="authenticator-description infoSection MuiBox-root emotion-8"
                       >
                         <h3
-                          class="MuiTypography-root MuiTypography-h3 authenticator-label emotion-25"
+                          class="MuiTypography-root MuiTypography-h3 authenticator-label no-translate emotion-25"
```
Comment (glenfannin-okta) [inline src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap:1712]: I don't think this should have the `no-translate` class. 🤔 

This string has a key in all 27 languages.

File: src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx [hunk 2]
```diff
@@ -142,7 +143,7 @@ const AuthenticatorButton: UISchemaElementComponent<{
           id={`${iconName}-label`}
           sx={{ fontSize: '1rem', margin: 0, textAlign: 'start' }}
           data-se="authenticator-button-label"
-          className="authenticator-label"
+          className="authenticator-label no-translate"
```
Comment (glenfannin-okta) [inline src/v3/src/components/AuthenticatorButton/AuthenticatorButton.tsx:146]: I think this is the root cause of each authenticator button label having the `no-translate` class. I think this should be conditional and only apply to things that cannot be translated like Yubikey / Okta verify / Symantec VIP / etc

**Review by glenfannin-okta — COMMENTED** [review]
Couple comments, but overall looks great! 

One additional comment, can you run this script `yarn run test:e2e:screenshots` locally and push the updated screenshots into the repo. I just want to make sure we keep those screenshots updated so we have a good reference.

File: src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap (hunk 1 above)
Comment (leemchale-okta) [reply src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap:1712]: In v2 it looks like it puts the `no-translate` class on all auth buttons [here](https://github.com/okta/okta-signin-widget/blob/2e253978624081b80e01d0433a7b0245fca1eb94/src/v2/view-builder/components/AuthenticatorVerifyOptions.js#L30)

Comment (glenfannin-okta) [reply src/v3/test/integration/__snapshots__/authenticator-verification-select-authenticator.test.tsx.snap:1712]: Oh it sure does, interesting. Ty!

**Review by glenfannin-okta — APPROVED** [review]

**Pull request merged**